src/core/Pagination/index.stories.tsx

```diff
@@ -0,0 +1,94 @@
+/* eslint-disable no-use-before-define */
+import { Args, Story } from "@storybook/react";
+import * as React from "react";
+import PaginationRaw from "./index";
+
+const Pagination = (props: Args): JSX.Element => {
+  const { pageSize, totalCount, siblingCount, truncateDropdown, sdsStyle } =
+    props;
+  const [currentPage, setCurrentPage] = React.useState(1);
+  return (
+    <PaginationRaw
+      sdsStyle={sdsStyle}
+      pageSize={pageSize}
+      onPageChange={(page) => {
+        setCurrentPage(page as number);
+      }}
```

Comment:
I prefer to avoid type assertion if possible. Does this satisfy the type checker?

```suggestion
      onPageChange={(page: number) => {
        setCurrentPage(page);
      }}
```

Reply:
Yup that worked. Thanks 👍🏻